git-d8ci

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Ubuntu / Debian traditionally use dash. The script should either not rely on any bash features or declare bash directly as done in the PR. The code would also not run with zsh or or tcsh, so this argument is moot.